1091. Shortest Path in Binary Matrix/solution.py

```diff
@@ -0,0 +1,42 @@
+class Solution(object):
+	# Given an n x n binary matrix grid, we return the shortest clear path in 
+	# the matrix, or -1 for no clear path
+
+	# A clear path is a path from the top-left cell to the bottom right cell
+	# such that:
+
+	 	# 1. all visited cells are 0
+	 	# 2. all adjacent cells are different and share an edge or corner
+
+
+	def findAdjacent(matrix, start_row, start_col):
```

Comment:
It looks like your leetcode is set to Python. You should select Python 3. One of the benefits of python 3 is type declaration for inputs and outputs. So for example, the definition of a function looks like this:

def shortestPathBinaryMatrix(self, grid: List[List[int]]) -> int: